int8 casualty_id
string[] image_path_list